lib/controller/controller.go

```diff
@@ -419,7 +420,17 @@ func NewProvisionController(
 	} else {
 		eventRecorder = broadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: fmt.Sprintf("%s %s %s", provisionerName, strings.TrimSpace(string(out)), string(identity))})
 	}
-
+	var CustomTermLimit time.Duration
+	LeaseTimeout, ok := os.LookupEnv("PROVISIONER_LEASE_TIMEOUT")
+	if !ok {
+		// TermLimit is the maximum duration that a leader may remain the leader
+		// to complete the task before it must give up its leadership.
+		// Defaults to 30 seconds.
+		CustomTermLimit = DefaultTermLimit
+	} else {
+		CustomTermLimit, _ = time.ParseDuration(LeaseTimeout)
+	}
+	glog.Infof("using customized provisioner termLimit seconds:%d", CustomTermLimit)
```

Comment:
can the message remove the word `customized`?

Reply:
Yes.